### DISCUSSION TITLE
Scale ToggleSwitch on Linux

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#10697] [Q&A] [answered]
My project needs to run on Linux FB and I'm looking to create a large ToggleSwitch that is easy to click on with your finger. On my WinUI platform I just set the Scale property and it works great. But on Linux this doesn't work (I get a warning that Scale isn't implemented). Is there another way to do this?

**Comment by jeromelaban** [comment, marked as answer]
See https://github.com/unoplatform/uno/issues/10674 for a workaround, where `TranslateTransform` and `ScaleTransform` can be used.

**Reply by HakanL** [reply]
Cool, that worked, thanks!

**Reply by HakanL** [reply]
Here's an example of what I did, maybe it'll help others:
```
<ToggleSwitch>
	<ToggleSwitch.RenderTransform>
		<ScaleTransform ScaleX="2.0" ScaleY="2.0"></ScaleTransform>
	</ToggleSwitch.RenderTransform>
</ToggleSwitch>
```

**Reply by jeromelaban** [reply]
Great! Also note that on the Framebuffer support, you can explicitly increase [the overall scaling](https://platform.uno/docs/articles/features/using-linux-framebuffer.html?q=framebu#dpi-scaling-support) (and ignore hardware specified values).

**Reply by HakanL** [reply]
That's cool. I'm aiming to have the WinUI head used for development, so it should be pixel-perfect, is there a similar scaling for WinUI so it can look the same?

**Reply by jeromelaban** [reply]
There isn't a global way of changing scaling that I know of on WinAppSDK, but maybe there's a compatibility setting in Win32 that exists to do that.
